src/plugins/validation/validator.js

```diff
@@ -21,7 +23,18 @@ class Validator extends Plugin {
 			};
 			const validData = this.validator.validate(target);
 			if (!validData) {
-				return this.validator.getErrors()[this.key];
+				const errors = this.validator.getErrors()[this.key];
+				if (isString(errors)) {
+					return errors;
+				} else if (isArray(errors) && isArray(this.value)) {
+					const result = errors.reduce((prev, curr, i) => {
+						if (curr !== null) {
+							prev.push(`${this.value[i]}: ${curr}`);
+						}
+						return prev;
+					}, []);
+					return result.join('\n');
```

Comment:
can we always return array of strings? so UI template can decide how to show this?

Reply:
Yes, we can. it would be better